string name
string type
float32 min
float32 max
float32 step
string value

